x-pack/plugins/security_solution/public/value_list/translations.ts

```diff
@@ -163,6 +156,10 @@ export const INFO_TOTAL_ITEMS = i18n.translate('xpack.securitySolution.listItems
 
 export const getInfoTotalItems = (listType: string) =>
   i18n.translate('xpack.securitySolution.listItems.searchBar', {
-    defaultMessage: 'Filter your data using KQL syntax - {listType}:*',
+    defaultMessage: 'Filter your data using KQL syntax, for example - {listType}:*',
```

Comment:
Here, the colon is more appropriate for separating the instruction from the example query.

```suggestion
    defaultMessage: 'Filter your data using KQL syntax, for example: {listType}:*',
```